### PR TITLE
Add  CL_0017002 (prostate neuroendocrine cell)

### LINKED ISSUE #1135: [NTR] Neuroendocrine cell of prostate epithelium
Note - please check that the term does not already exist (check OLS: https://www.ebi.ac.uk/ols/ontologies/cl)

**Preferred term label:**
Neuroendocrine cell of prostate epithelium

**Synonyms**

**Definition (free text, please give PubMed ID in format PMID:XXXXXX)**
Neuroendocrine cell (CL:0000165) in the prostate
* PMID:33842835
* PMID:31390564
* PMID:32497356

**Parent cell type term (check the hierarchy here https://www.ebi.ac.uk/ols/ontologies/cl):**
CL:0002231

**What is the anatomical structure that the cell is a part of? Please check Uberon: https://www.ebi.ac.uk/ols/ontologies/uberon**
UBERON:0000428

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0001-7772-9578

**Any additional notes or concerns**

### PR DESCRIPTION
**Pull request by lubianat**
Closes #1135 .

Weirdly modifying all  (oboInOwl:hasDbXref "GOC:tfm")  to (oboInOwl:hasDbXref "GOC:tfm"^^xsd:string).

(@matentzn this is the PR)

### CODE REVIEW

**Comment by matentzn** [comment]
- [ ] @lubianat please assign suitable reviewer

File: src/ontology/cl-edit.owl [hunk 1]
```diff
@@ -23174,6 +23174,17 @@ AnnotationAssertion(rdfs:label obo:CL_0017001 "splanchnic mesodermal cell")
...
+# Class: obo:CL_0017002 (prostate neuroendocrine cell)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:33842835"^^xsd:string) Annotation(oboInOwl:hasDbXref "http://orcid.org/0000-0001-7772-9578"^^xsd:string) obo:IAO_0000115 obo:CL_0017002 "A neuroendocrine cell that is part of the prostate epithelium.")
+AnnotationAssertion(oboInOwl:created_by obo:CL_0017002 <http://orcid.org/0000-0003-2473-2313>)
+AnnotationAssertion(oboInOwl:hasExactSynonym obo:CL_0017002 "neuroendocrine cell of prostate epithelium")
+AnnotationAssertion(oboInOwl:hasExactSynonym obo:CL_0017002 "neuroendocrine cell of the prostate")
+AnnotationAssertion(rdfs:label obo:CL_0017002 "prostate neuroendocrine cell")
+EquivalentClasses(obo:CL_0017002 ObjectIntersectionOf(obo:CL_0000165 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0000428)))
+SubClassOf(obo:CL_0017002 obo:CL_0000165)
+SubClassOf(obo:CL_0017002 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0000428))
```
Comment (dosumis) [inline src/ontology/cl-edit.owl:23186]: No need to duplicate EquivalentTo axiom clauses as SubClassOf axioms

**Review by dosumis — CHANGES_REQUESTED** [review]
Minor change requested to remove redundancy.  Please see comment

File: src/ontology/cl-edit.owl (hunk 1 above)
Comment (lubianat) [reply src/ontology/cl-edit.owl:23186]: @dosumis is anything still left here to do?
 I am sorry for the long delay by the way.

**Comment by nicolevasilevsky** [comment]
@dosumis @lubianat I fixed the conflict on this, is there any other action needed?

**Review by shawntanzk — APPROVED** [review]

**Comment by shawntanzk** [comment]
@lubianat number of PRs getting a bit long, decided to go in and see if there's anything I can do a review on to move along :)

**Review by dosumis — APPROVED** [review]

**Comment by dosumis** [comment]
Looks good, but conflicts again I'm afraid.  Presumably due to consecutive IDs with recently merged term.

**Comment by shawntanzk** [comment]
I've fixed the conflicts, @lubianat will leave you to merge it :)

**Pull request merged**